src/validation.cpp

```diff
@@ -5795,9 +5795,7 @@ bool LoadExternalBlockFile(const CChainParams& chainparams, FILE* fileIn, CDiskB
     } catch (const std::runtime_error& e) {
         AbortNode(std::string("System error: ") + e.what());
     }
-    if (nLoaded > 0)
-        LogPrintf("Loaded %i blocks from external file in %dms\n", nLoaded, GetTimeMillis() - nStart);
-    return nLoaded > 0;
+    LogPrintf("Loaded %i blocks from external file in %dms\n", nLoaded, GetTimeMillis() - nStart);
```

Comment:
Leave the boolean return in; and "return false" on the ShutdownRequested().  While the calling routines don't check the success of calls to this routine currently; it is something that may get addressed at a later date (as well as better error handling within `LoadExternalBlockFile()`.  

Rest of the code looks good.  Just this part that I prefer is left as is; or perhaps refined so that
```
if (nLoaded > 0) {
    LogPrintf("...)
} else {
    LogPrintf("ERROR: ...
}

Reply:
nack, there are only 3 calls of the functions and none used the return value, and the function is not expected to fail per "the try/catch". I really think the commit is better as is without the confusing bool.

Reply:
I agree with @CaveSpectre11, just leave it in for his reasons stated that it likely will get addressed at a later date with better error handling. Out of scope to remove it.

Reply:
I disagree. we'll go with democracy, undoing that bit.